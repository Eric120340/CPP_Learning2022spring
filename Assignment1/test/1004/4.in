3 3 2
15 46 39 
24 6 48 
12 23 3 
