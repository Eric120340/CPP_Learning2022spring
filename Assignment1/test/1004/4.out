1797 1863 2910 
1080 2244 1368 
768 759 1581 
